### ISSUE TITLE
ASCII formatting does not align

### ISSUE THREAD

**Issue by justinwoo** [dhall-lang/dhall-haskell#832]
Probably already reported, but for reference:

```hs
  \(dependencies : List Text)
-> \(repo : Text)
-> \(version : Text)
->   { dependencies = dependencies, repo = repo, version = version }
   : ./Package.dhall
```

versus unicode (which is already misaligned for me due to arrows being double-width for fonts i use, e.g. Noto Sans Mono CJK JP):

```hs
  λ(dependencies : List Text)
→ λ(repo : Text)
→ λ(version : Text)
→   { dependencies = dependencies, repo = repo, version = version }
  : ./Package.dhall
```

**Comment by justinwoo** [comment]
example of what unicode syntax looks like for me:

![screenshot from 2019-02-27 18-40-13](https://user-images.githubusercontent.com/2396926/53506689-31317480-3abf-11e9-88bb-69bbd2d32698.png)

**Comment by justinwoo** [comment]
This is on dhall-1.20.1, not master.

**Comment by f-f** [comment]
It looks like this is fixed in #831:

Installing dhall from that branch and putting your snippet in `test.dhall`:
```
$ cat test.dhall | dhall --ascii
    \(dependencies : List Text)
->  \(repo : Text)
->  \(version : Text)
->  { dependencies = dependencies, repo = repo, version = version }
```

**Comment by Gabriel439** [comment]
Yep!  That pull request fixes this issue
